webrtc/peer.go

```diff
@@ -147,6 +147,7 @@ func (p *Peer) AddCandidates(session *webrtc.SessionDescription, candidates []*w
 		}
 
 		p.answer = &answer
+		// cborys: What are these udp listeners? I don't see them here?
 		// Sets the LocalDescription, and starts our UDP listeners
 		err = p.connection.SetLocalDescription(answer)
```

Comment:
Setting the local description triggers the connection establishment here. So the data channel will be opened and in `p.gameDataChannel.OnOpen` we register the udp listener